.gitignore

```diff
@@ -0,0 +1,2 @@
+/result
```

Comment:
why do these start from root? unless i misunderstand how paths in .gitignore. regardless, even if this works, it's confusing because based on this i would expect to be able to
```
cd /
ls home/chessai # just to illustrate where we are relatively
cd result
```
on my machine.

this should probably be
```
result
result-*
build/result
build/result-*
```
the result-* paths are for doc outputs and other things.